pyuvdata/uvdata/tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright (c) 2021 Radio Astronomy Software Group
+# Licensed under the 2-clause BSD License
+
+"""Testing fixtures for UVData."""
+import os
+
+import pytest
+
+from pyuvdata.data import DATA_PATH
+from pyuvdata import UVData
+
+
+@pytest.fixture(scope="session")
+def hera_uvh5_main():
+    # read in test file for the resampling in time functions
+    uv_object = UVData()
+    testfile = os.path.join(DATA_PATH, "zen.2458661.23480.HH.uvh5")
+    uv_object.read(testfile)
+
+    yield uv_object
+
+    # cleanup
+    del uv_object
+
+    return
```

Comment:
Is this fixture defined somewhere else currently? I can't help but think this feels familiar :thinking:

Reply:
yes, I thought I moved it here so I could use it across multiple files, but maybe I just copied it. I'll check.

Reply:
Yep, that's what happened. I just pushed up the removal from test_uvdata.py. Thanks for catching that!